fluence.yaml

```diff
@@ -5,14 +5,19 @@
 # Documentation: https://github.com/fluencelabs/fluence-cli/tree/main/docs/configs/fluence.md
 
 version: 2
-aquaInputPath: src/aqua/main.aqua
+aquaInputPath: src/aqua/decider/poll.aqua
+aquaImports:
+  - src/aqua
+  - src/aqua/decider
+  - src/aqua/chain
+  - src/aqua/fluence
```

Comment:
@shamsartem let's talk about that :)

Reply:
Let's talk. But I don't understand the problem and why it looks like this